guardian/recipes-backend
# CAPI interface

## What does this change?

Implements https://trello.com/c/JBnAeWBs

## How to test

- Functions are tested with automated tests
- If you deploy it, the function should no longer fail when updates come through but print a single line to the log for each update (takedowns still cause errors)
- Retrievable update acquisition is now implemented

## How can we measure success?

Movin' along
